libs/langchain/langchain/hub.py

```diff
@@ -0,0 +1,50 @@
+from typing import Any, Dict, Optional
+
+from langchainhub import Client
```

Comment:
if this is optional dependency probably better to import within relevant functions (ie within _get_client). makes it safer to import code from this module

Reply:
Done! Would love a quick check of the type checking bit